Add Documentation for "User Membership" Methods (#108)
CitrineInformatics/citrine-python: docs/source/getting_started/projects_and_users.rst

> =======================

Reviewer: Change to one sentence per line

Projects and Users
=======================

Overview
--------

> Projects are containers for other resources, like datasets and modules. Access rights on resources inside a Project are

Reviewer: ```suggestion
Projects are containers for other resources, such as datasets and modules. Access rights on resources inside a Project are
```
Previous wording could be interpreted as saying that projects are like datasets and modules in that they are all containers for other resources

managed, granted, and revoked at the Project level.

> Users are literal users of the Citrine Platform. Users cannot directly access resources, but must instead be added to

Reviewer: ```suggestion
Users are individuals using the Citrine Platform. Users cannot directly access resources, but must instead be added to
```

Projects. Through their membership in Projects, Users can access resources.

This library presents several methods for managing Projects, Users, and user membership in Projects.

Listing Users in a Project
----------------------

Users in a Project can be listed using the :func:`~citrine.resources.project.Project.list_members` method.
The ProjectMember array returned from this method has the user's role in the project as well as a copy
of the User and Project objects.

Add User to a Project
--------------------

Users can be added to a Project. They will be granted READ access to resources in the Project and will be given the

> role "MEMBER". This can be accomplished with the :func:`~citrine.resources.project.Project.add_user` method.

Reviewer: ```suggestion
role "MEMBER". This is accomplished with the :func:`~citrine.resources.project.Project.add_user` method.
```

Remove User from a Project
-----------------------

> Users can also be removed from a Project. This can be achieved with the

Reviewer: ```suggestion
Users can also be removed from a Project. This is achieved with the
```
